dapps/onboarding.vue/src/components/twin-sign-up/twin-sign-up.ts

```diff
@@ -441,7 +441,7 @@ export default class TwinSignUp extends mixins(SignUp) {
       }, 2000);
     } catch (ex) {
       // reset all steps of proile creation
-      dappBrowser.utils.log(ex.message, 'error');
+      utils.devLog(ex.message, 'error');
```

Comment:
devLog ? Wouldn't it be better to use something like log level "debug" and enable a lower log level in dev mode?